CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.0)
+
+if (APPLE)
+  option(BUILD_FRAMEWORK "Build a Mac OS X framework instead of a shared library" OFF)
+  set(CMAKE_OSX_ARCHITECTURES "i386;x86_64")
+endif ()
+
+#include(CheckStructHasMember)
+#CHECK_STRUCT_HAS_MEMBER("struct dirent" d_type dirent.h HAVE_STRUCT_DIRENT_D_TYPE LANGUAGE CXX)
+
+file(GLOB SOURCES src/*.cpp)
+file(GLOB HEADERS src/*.h)
+file(GLOB API_HEADERS include/*.h)
+add_library(SAASound SHARED ${SOURCES} ${HEADERS} ${API_HEADERS})
+
+set_target_properties(SAASound PROPERTIES
+  VERSION 3.2
+  SOVERSION 3
+  CXX_STANDARD 11)
+
+target_include_directories(SAASound PRIVATE
+  ${CMAKE_BINARY_DIR}
+  include)
+
+if (APPLE AND BUILD_FRAMEWORK)
+  set_target_properties(SAASound PROPERTIES
+    OUTPUT_NAME SAASound
+    FRAMEWORK TRUE
+    FRAMEWORK_VERSION C
+    MACOSX_FRAMEWORK_IDENTIFIER org.rebuzz)
```

Comment:
I guess I don't know the impact/relevance of having "org.rebuzz" here  :-)  Happy to leave it if there's no negative implication.

Reply:
It's just a unique identifier for the framework, which is usually the domain in reverse.  May be worth updating in some future commit, but harmless to be your old domain in the meantime. Not actually sure what it's used for by the OS!